tool/build_release.sh

```diff
@@ -17,16 +17,16 @@ DEVTOOLS_DIR="${TOOL_DIR}/.."
 FLUTTER_DIR="`pwd`/flutter-sdk"
 PATH="$FLUTTER_DIR/bin":$PATH
 
-REQUIRED_FLUTTER_VERSION=$(<"${DEVTOOLS_DIR}/flutter-version.txt")
+REQUIRED_FLUTTER_VERSION=`dart $TOOL_DIR/bin/repo_tool.dart latest-flutter-candidate | tail -n 1`
```

Comment:
is the `|tail - n 1 necessary` here? it looks like the command would only ever print one line

Reply:
the command may have to call `flutter pub get`, in which case the stdout will get clogged up with all that output